src/main/java/org/openRealmOfStars/game/Game.java

```diff
@@ -1580,8 +1580,11 @@ public void showVotingSelectionView() {
    */
   public void showCredits() {
     try {
+      var is = Game.class.getResourceAsStream("/AUTHORS.md");
```

Comment:
This stream is not closed. Stream reading should be done try-with resources. Personally I haven't used var yet, but my first impressions are it makes code reading a bit harder. It works when you have IDE around, but when looking via Github one cannot immediately see the class.